backend/functions/lib/adapters/gateways/cases.dxtr.gateway.ts

```diff
@@ -37,9 +37,46 @@ export default class CasesDxtrGateway implements CasesInterface {
   async getChapter15Cases(
     context: ApplicationContext,
     options: { startingMonth?: number },
-  ): Promise<Chapter15CaseInterface[]> {
-    const doChapter12Enable = context.featureFlags['chapter-twelve-enabled'];
-    const rowsToReturn = doChapter12Enable ? '10' : '20';
```

Comment:
Are we no longer using the chapter-twelve-enabled flag?  And what about the number of rows to return?

Reply:
I noticed this too and this should not be. I must have inadvertently checked in changes I didn't mean to or something went wrong with a rebase. This should be fixed now.